homeassistant/components/sun/strings.json

```diff
@@ -27,6 +27,15 @@
       "solar_azimuth": { "name": "Solar azimuth" },
       "solar_elevation": { "name": "Solar elevation" },
       "solar_rising": { "name": "Solar rising" }
+    },
+    "binary_sensor": {
+      "solar_rising": {
+        "name": "Solar rising",
+        "state": {
+          "on": "Rising",
+          "off": "Falling"
```

Comment:
Could use a second opinion between "Setting" and "Falling" as a translation here. "Setting" sounds perhaps natural during the day, but once the sun is below the horizon, I couldn't decide if "Setting" was still correct after the sun had already set 🤷 .

Reply:
Allright lets go for that then.